spec/factories/services.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :service do
+    identifier ""
+    label "MyString"
+  end
```

Comment:
For all of the factories it would be nice if there were real looking examples for each field instead of the blank string or "MyString" defaults.